src/main/java/com/uid2/admin/util/InputUtil.java

```diff
@@ -0,0 +1,85 @@
+package com.uid2.admin.util;
+
+public class InputUtil {
```

Comment:
is this class copied? Can we move it to shared instead?

Reply:
Yep, but just one method copied from the larger "InputUtil" class in uid2-client-java. Moving normalization logic to shared is part of [UID2-1206](https://atlassian.thetradedesk.com/jira/browse/UID2-1206). Once that is moved, the only change needed in Admin should be to delete InputUtil and change the import statement to shared. Assuming this contact is for human consumption only I'd say normalization is not necessary, however could be useful for catching typos.

Reply:
1206 might not get done (see comments there). If you look at uid2-shared commits, there have been a few changes (eg by @cYKatherine ) where we have moved code to shared so it can be used by >1 project. Since we don't want to move all of InputUtil, we could use a different class in shared and make InputUtil depend on that

Reply:
Discussed in slack, let's remove normalization for now

Reply:
removed